JS/light-dark-mode.js

```diff
@@ -117,10 +146,15 @@ const checkSelectedMode = (mode) => {
     changeToDarkMode();
     btnDarkMode.style.display = "none";
     btnLightMode.style.display = "block";
-  } else {
-    changeToDarkMode();
-    btnDarkMode.style.display = "none";
-    btnLightMode.style.display = "block";
   }
-};
-checkSelectedMode(localStorage.mode);
+}
+btnLightMode.addEventListener("click", () => {
+  localStorage.setItem('theme', 'light')
+  changeTheme(localStorage.getItem('theme'))
+});
+
+btnDarkMode.addEventListener("click", () => {
+  localStorage.setItem('theme', 'dark')
```

Comment:
Really liked this method of using `localStorage` to keep a track of the mode selected. Got to learn something new, will surely learn more and implement, Very well done 👍